### PR TITLE
Default directory & formatting changes

### PR DESCRIPTION
**Pull request by dieunity**
User put into their opened directory (OpenFolderButton.jsx)

### CODE REVIEW

**Review by MaxWeisen — APPROVED** [review]
Good stuff. Getting us closer to our MVP!!

**Review by JIB3377 — APPROVED** [review]
Great job!

File: src/components/OpenFolder/OpenFolderButton.jsx [hunk 1]
```diff
@@ -36,17 +37,19 @@ const OpenFolder = () => {
...
       dispatchToGlobal(setProjectFilePath(directoryPath));
       dispatchToGlobal(createFileTree(generateFileTreeObject(directoryPath)));
       dispatchToGlobal(loadProject('load'));
       dispatchToGlobal(setTestCase(''));
       if (!isTestModalOpen) dispatchToGlobal(toggleModal());
       if (!isFileDirectoryOpen) dispatchToGlobal(toggleFileDirectory());
+
+      // Re-direct terminal directory to user selected directory
+      ipc.send('terminal.toTerm', `cd ${directoryPath}\n`);
     }
```
Comment (mhmaidi789) [inline src/components/OpenFolder/OpenFolderButton.jsx:53]: Great Idea automatically setting terminal directory to profile file path.

**Review by mhmaidi789 — APPROVED** [review]

File: src/components/OpenFolder/OpenFolderButton.jsx (hunk 1 above)
Comment (MaxWeisen) [reply src/components/OpenFolder/OpenFolderButton.jsx:53]: I agree!

**Pull request merged**